data/shared/citizen/scripting/v8/timer.js

```diff
@@ -85,10 +85,18 @@
         animationFrames[animationFrames.length] = callback;
     }
 
-    function setInterval(callback, interval) {
+    function setInterval(callback, interval, ...argsForCallback) {
```

Comment:
It would be great if you'd also add support for the args passing to the `setTimeout` method as well.

Reply:
That's why I added argsForCallback

Reply:
setTimeout has already:

```js
test(arg) {
   console.log(arg); // Print exemple
}

setTimeout(test, 100, "exemple");
```

Reply:
Aww, yeah, you're right, it's already there, my bad.